### ISSUE TITLE
Mandatory update popup won't show after closing the app

### ISSUE THREAD

**Issue by Kaizodo** [larryaasen/upgrader#426]
I am not sure this is a bug or a feature but here is how you can reproduce this situation 

On a device if i set 

```dart
UpgradeAlert(
      showIgnore: false,
     showLater: false,
child : Conainer())
```

this will disable closing the popup so it will make update mendatory however if user closes the app and reopens it popup did not show again and stays  shut for default 3 days with this default setting in Upgrader class 
```dart 
  Duration durationUntilAlertAgain = const Duration(days: 3),
``` 

After 3 days later i will/should prompt again (i haven't test it)

so i think it if update is mendatory like showIgnore is false and showLater is false we should skip the later thing and keep update popup open until app is updated.

I have hope we are not saving ignored version when closing or starting the app otherwise ```alreadyIgnoredThisVersion()``` will return true and popup won't show even if we set  durationUntilAlertAgain  to Duration.zero

**Comment by Kaizodo** [comment]
I thought its a version related issue (Major, minor ,patch) so i deployed a major version of my app and plugin still did not popup the upgrade dialoge this works randomly sometimes it opens popup and sometimes it does not.

**Comment by erdzan12** [comment]
Yes this should be something one could configure based on the needs, like to force users to update the app

**Comment by phamgiaminh0112** [comment]
I have corrected this error and created a pull request. In that you just need to remove the isTooSoon() condition in file src/upgrader.dart line number 306. And don't forget to set showIgnore = false and showLater = false if you don't want users to skip the update.

**Comment by larryaasen** [comment]
Did you try using `durationUntilAlertAgain` with `Upgrader` and setting it to `Duration(seconds: 0)`?

**Comment by deepbhikadiya** [comment]
Is there anything update??
I want to show this upgrade package everytime when user open the app if user not update app yet.
@phamgiaminh0112
